lib/api/new.coffee

```diff
@@ -46,6 +46,12 @@ class New
 
     opts =
       locals: opts.overrides ? {}
+      questionnaire: (questions, skip) ->
+        W.promise (resolve, reject) ->
+          qs = []
+          for question in questions
+            qs.push(question) unless !_.contains(skip, question.name)
```

Comment:
I think you mean `qs.push(question) unless _.contains(skip, question.name)`  (gets rid of the double negative)